integration-tests/cassandra-client/pom.xml

```diff
@@ -0,0 +1,162 @@
+                    <dependency>com.datastax.oss.quarkus:cassandra-quarkus-integration-tests-rpkgtests</dependency>
+                  </dependenciesToScan>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+    <profile>
+      <id>native</id>
+      <activation>
+        <property>
+          <name>native</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-maven-plugin</artifactId>
+            <version>${quarkus.version}</version>
+            <executions>
+              <execution>
+                <id>native-image</id>
+                <goals>
+                  <goal>native-image</goal>
+                </goals>
+                <configuration>
+                  <appArtifact>com.datastax.oss.quarkus:cassandra-quarkus-integration-tests:${quarkus-cassandra-client.version}</appArtifact>
+                    <reportErrorsAtRuntime>false</reportErrorsAtRuntime>
+                    <cleanupServer>true</cleanupServer>
+                    <enableHttpUrlHandler>true</enableHttpUrlHandler>
+                    <enableServer>false</enableServer>
+                    <dumpProxies>false</dumpProxies>
+                    <graalvmHome>${graalvmHome}</graalvmHome>
+                    <disableReports>true</disableReports>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-failsafe-plugin</artifactId>
+            <configuration>
+              <dependenciesToScan>
+                <dependency>com.datastax.oss.quarkus:cassandra-quarkus-integration-tests-rpkgtests</dependency>
+              </dependenciesToScan>
+            </configuration>
+            <executions>
+              <!-- VM tests -->
+              <execution>
+                <id>default</id>
+                <goals>
+                  <goal>integration-test</goal>
+                  <goal>verify</goal>
+                </goals>
+                <configuration>
+                  <excludedGroups>native</excludedGroups>
+                  <summaryFile>${project.build.directory}/failsafe-reports/failsafe-summary-vm.xml</summaryFile>
+                </configuration>
+              </execution>
+              <!-- Native tests -->
+              <execution>
+                <id>native-tests</id>
+                <goals>
+                  <goal>integration-test</goal>
+                  <goal>verify</goal>
+                </goals>
+                <configuration>
+                  <groups>native</groups>
+                  <excludes>
+                    <!-- this test is broken in 1.0.0-alpha1 -->
+                    <exclude>**/CassandraMetricsNativeIT.java</exclude>
```

Comment:
This test will be fixed for the next alpha, see https://github.com/datastax/cassandra-quarkus/issues/96.